documentation/openapi-authoring-automated-guidelines.md

```diff
@@ -1015,13 +1016,13 @@ Links: [Index](#index) | [Error vs. Warning](#error-vs-warning) | [Automated Rul
 
 Links: [Index](#index) | [Error vs. Warning](#error-vs-warning) | [Automated Rules](#automated-rules) | [RPC](#rpc-violations): [Errors](#rpc-errors) or [Warnings](#rpc-warnings) | [SDK](#sdk-violations): [Errors](#sdk-errors) or [Warnings](#sdk-warnings)
 
-### <a name="R2064" />R2064 LicenseMissing
-**Output Message**: Please provide correct licensing information here. Acceptable value: "name": "MICROSOFT\_MIT\_NO\_VERSION"
+### <a name="R2065" />R2065 LicenseHeaderMustNotBeSpecified
+**Output Message**: License header must not be specified in the OpenAPI document.
 
-**Description**: Each OpenAPI json document must contain the license object in its `info` section which must set the name to `MICROSOFT\_MIT\_NO\_VERSION`. This is necessary for AutoRest to insert the appropriate headers in the generated code.
+**Description**: `x-ms-code-generation-settings` must not have the header section specified in the OpenAPI documents since each SDK has a different licensing header, this information must be provided either from the command line or the configuration file.
 
-**Why the rule is important**: Missing or invalid license information can cause incorrect or empty headers in the generated code.
+**Why the rule is important**: License information must not be specified in the OpenAPI document since different SDKs have different headers.
```

Comment:
I'd reword that a bit to make clear that it's about the license of *generated* stuff, since the license of the Swagger (`info.license`) is of course something you can specify